parity-dev-node/instant-seal-chain-spec.json

```diff
@@ -4,11 +4,24 @@
         "instantSeal": null
     },
     "params": {
-        "accountStartNonce": "0x0",
         "gasLimitBoundDivisor": "0x400",
+        "accountStartNonce": "0x0",
         "maximumExtraDataSize": "0x20",
         "minGasLimit": "0x1",
-        "networkID" : "0x11"
+        "networkID" : "0x11",
+        "eip150Transition": "0x0",
+        "eip160Transition": "0x0",
+        "eip161abcTransition": "0x0",
+        "eip161dTransition": "0x0",
+        "eip649Reward": "0x29A2241AF62C0000",
+        "eip100bTransition": "0x0",
+        "eip649Transition": "0x0",
+        "eip98Transition": "0xffffffffffffffff",
+        "eip140Transition": "0x0",
+        "eip211Transition": "0x0",
+        "eip214Transition": "0x0",
+        "eip155Transition": "0x0",
+        "eip658Transition": "0x0"
```

Comment:
What are these settings for?

Reply:
These enable the various hard-fork changes at block 0.  "Byzantium" is really a rollup of a bunch of changes, as is Homestead, etc.  Parity tracks them each separately, rather than all in one block.